e2e/app/element/base-element.ts

```diff
@@ -84,10 +85,10 @@ export default class BaseElement {
    *  return await handle.jsonValue();
    */
   async getProperty(propertyName: string): Promise<unknown> {
-    if (this.element == null) {
+    if (this._element == null) {
```

Comment:
Why not make this a `string?`-valued function instead of throwing? You can always throw later if the property name was critical.